_conferences/kotlin-in-boston-everywhere-2019.md

```diff
@@ -0,0 +1,8 @@
+---
+name: "Kotlin in Boston Everywhere - DevFest"
```

Comment:
Why is this suffixed with "DevFest"? If it's a GDG DevFest that should probably be its only name to match the others. There's also some Kotlin Everywhere events on the list have been using just "Kotlin/Everwhere" as their name.

Reply:
that's just what the conference organizers ended up settling on. it is technically both a Kotlin/Everywhere Event and a GDG Devfest. not my first choice...

is this going to prevent this being posted?

Reply:
Nope. I'm just trying to understand the event a bit more. It's clearly large enough to warrant inclusion given that it occurs over two days and has multiple tracks. Historically there's been smaller, half-day Kotlin/Everywhere events that didn't really classify as conferences proposed.